internal/stats/stats.go

```diff
@@ -1,16 +1,30 @@
 package stats
 
 import (
-	"fmt"
+	"github.com/NodeFactoryIo/hactar-daemon/internal/hactar"
+	"github.com/NodeFactoryIo/hactar-daemon/internal/lotus/services"
+	"github.com/NodeFactoryIo/hactar-daemon/internal/url"
 	log "github.com/sirupsen/logrus"
 	"github.com/spf13/viper"
 	"strconv"
 	"time"
 )
 
 func SubmitNewStatsReport() bool  {
-	// TODO collect all statistic and send it to backend
-	fmt.Println("Collecting stats and sending report....")
+	client := hactar.NewClient(nil)
+	lotusService := services.NewLotusService(nil, nil)
+
+	nodeUrl := url.GetUrl()
+	actorAddress, err := lotusService.GetMinerAddress()
```

Comment:
Is this a kind of health check? Good idea but separate this in another file as it will be replace later probably.

Reply:
@morrigan 
I will refractor this even more in some next task because I think that this pair `(nodeUrl, actorAddress)` should be in some kind of structure because it is used always as pair.